karate-core/src/main/java/com/intuit/karate/core/ScenarioBridge.java

```diff
@@ -229,6 +229,17 @@ public Object callSingle(String fileName, Object arg) throws Exception {
         }
     }
 
+    public Object callonce(String filePath) {
+        return this.callonce(filePath, false);
+    }
+
+    public Object callonce(String filePath, boolean sharedScope) {
```

Comment:
actually shared scope will always return null - because it just resets all the variables (and fires some startup routines all over again). which is another reason why i feel this may be a premature optimization, graal js is fast. also note that we cache the karate-config.js in memory (see `Suite`) for the entire test run in the new version

anyway I'll double check and most likely will change this signature to return void